pkg/grpc/api.go

```diff
@@ -383,39 +384,37 @@ func (a *api) CallActor(ctx context.Context, in *internalv1pb.InternalInvokeRequ
 	return resp.Proto(), nil
 }
 
-func (a *api) PublishEvent(ctx context.Context, in *runtimev1pb.PublishEventRequest) (*emptypb.Empty, error) {
+func (a *api) validateAndGetPubsubAndTopic(pubsubName, topic string, reqMeta map[string]string) (pubsub.PubSub, string, string, bool, error) {
```

Comment:
Please either use named returns (like `thepubsub pubsub.PubSub, pubsubName string, topic string, isRawPayload bool, err error`) or add a comment explaining what the return values are for. Makes it easier for people reading the code in the future.

Reply:
Added a comment ...